back/CinemaReservation.Database/dbo/Tables/Seances.sql

```diff
@@ -0,0 +1,15 @@
+﻿CREATE TABLE [dbo].[Seances] (
+    [Id]            INT      IDENTITY (1, 1) NOT NULL,
+    [DateTime]      DATETIME NOT NULL,
+    [FilmId]        INT      NOT NULL,
+    [HallId]        INT      NOT NULL,
+    CONSTRAINT [PK_Seances_Id] PRIMARY KEY CLUSTERED ([Id] ASC),
+    CONSTRAINT [FK_Seances_FilmId_Films_Id] FOREIGN KEY ([FilmId]) REFERENCES [dbo].[Films] ([Id]),
+    CONSTRAINT [FK_Seances_HallId_Halls_Id] FOREIGN KEY ([HallId]) REFERENCES [dbo].[Halls] ([Id])
+);
+
+
+GO
+CREATE UNIQUE NONCLUSTERED INDEX [IX_Seances_HallId_DateTime]
```

Comment:
Although this constraint is locally valid it is useless in the same time because DateTime can be passed with millisecond precision